components/bitly/actions/find-bitlink/find-bitlink.mjs

```diff
@@ -0,0 +1,92 @@
+import { ConfigurationError } from "@pipedream/platform";
+import { formatDeepLink } from "../../common/common.utils.mjs";
+import bitly from "../../bitly.app.mjs";
+
+export default {
+  key: "bitly-find-bitlink",
+  name: "Retrieve a Bitlink",
+  description: "Returns information for the specified link.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly,
+    bitlink: {
+      type: "string",
+      description: "Bitlink url",
+    },
+    createBitlinkIfNotFound: {
+      label: "Create a new bitlink if the bitlink is not found?",
+      description:
+        "Create a new bitlink if no record is found for the specified bitlink.",
+      type: "string",
+      options: ["Yes", "No"],
+      default: "No",
+      reloadProps: true,
+    },
+  },
+  async additionalProps() {
+    const props = {};
+    if (this.createBitlinkIfNotFound === "Yes") {
+      props["long_url"] = {
+        type: "string",
+        description: "URL to shorten",
+      };
+      props["title"] = {
+        type: "string",
+        optional: true,
+        description: "Bitlink title",
+      };
+      props["tags"] = {
+        type: "string[]",
+        optional: true,
+      };
+      props["domain"] = {
+        type: "string",
+        description:
+          "Bitlinks that contain deeplinks configured with a custom domain",
+        optional: true,
+      };
+      props["group_guid"] = {
+        type: "string",
+        optional: true,
+        description: "Group GUID",
+      };
+      props["deeplinks"] = {
+        type: "string[]",
+        optional: true,
+        description: `Provide an object. Each object should represent a row. (e.g. {"app_id":"com.bitly.app","app_uri_path": "/store?id=123456","install_url": "https://play.google.com/store/apps/details?id=com.bitly.app&hl=en_US","install_type": "promote_install" })`,
+      };
+    }
+    return props;
+  },
+  async run({ $ }) {
+    let bitlinkDetail;
+    try {
```

Comment:
There is any specific reason for creating a Bitlink here? I mean, If this is just a find. I don't think we need to handle the creation here. The user can create a workflow by itself if the link is not found using the `create-bitlink`. In my view, we should not handle this here

Reply:
I also understand, since we have create-bitlink already.
However, the ticket stated this 
**Find Bitlink | Finds an existing Bitlink in your account. Optionally, creates one if none are found**

Reply:
Okay, In this case, you can ignore my comment